poc/components/PocButton.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Text, View, TouchableHighlight } from 'react-native';
+import styles from '../styles';
+
+export default function PocButton(props) {
+    return (
+        <View>
+            {/* TODO: invoke function passed in on state */}
+            <TouchableHighlight style={styles.button} onPress={props.onPress}>
+                <View>
```

Comment:
Why do we need this extra `View` tag wrapping the `Text`?

Reply:
I suppose not, good call